section_faiss_vector_index.go

```diff
@@ -657,7 +601,7 @@ type indexContent struct {
 
 type vecInfo struct {
 	vec    []float32
-	docIDs *roaring.Bitmap
+	docIDs uint64
```

Comment:
You changed the type but left the name as it were - change to docID and leave it at uint32.